### ISSUE TITLE
[Component] [checkbox] el-checkbox native checkbox value does not change on @change

### ISSUE THREAD

**Issue by wldrve** [element-plus/element-plus#10910]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.39`
- Element Plus Version: `2.2.26`
- Browser / OS: `Chrome 107.0.5304.107`
- Build Tool: `esbuild`

### Related Component

- `el-checkbox`

### Reproduction Link

[Link]()

### Steps to reproduce

El-Checkbox model-value binding does not work in conjuction with original input type value attribute.

1. Create el-checkbox component with boolean v-model
2. Check <input class="el-checkbox__original"... > value attribute 

### What is Expected?

Original checkbox value should be the same as binding v-model

### What is actually happening?

I presume because the checkbox component is wrapped in a label, the original input value is the parent's label value.

### Additional comments

If I am wrong and do not understand the usage of the component, please let me know how to change the original checkbox's value.

**Comment by itsalimanuel** [comment]
Can you share your code?
just add it to [demo](https://element-plus.run/)

**Comment by wldrve** [comment]
> Can you share your code? just add it to [demo](https://element-plus.run/)

https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgb25Nb3VudGVkIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBjaGVja2JveFZhbHVlID0gcmVmKGZhbHNlKTtcbmNvbnN0IGNoZWNrYm94UmVmID0gcmVmKG51bGwpO1xubGV0IG9yaWdpbmFsVmFsdWUgPSByZWYobnVsbCk7XG5cbm9uTW91bnRlZCgoKSA9PiB7XG4gIG9yaWdpbmFsVmFsdWUgPSBjaGVja2JveFJlZi52YWx1ZS4kZWwuY2hpbGRyZW5bMF0uY2hpbGROb2Rlc1swXS5fdmFsdWVcbn0pO1xuXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtY2hlY2tib3ggcmVmPVwiY2hlY2tib3hSZWZcIiB2LW1vZGVsPVwiY2hlY2tib3hWYWx1ZVwiIGxhYmVsPVwiT3B0aW9uIDFcIiAvPiBcbiAgPGJyLz48YnIvPlxuXG4gIHt7IGNoZWNrYm94VmFsdWUgfX0gPGJyPjxicj5cbiAge3sgb3JpZ2luYWxWYWx1ZSB9fVxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==

Here's the demo, please click on the checkbox to change it's value, to see the original input's value. I'm not used to writing TypeScript, so please bare that in mind. Thanks!

**Comment by wldrve** [comment]
![image](https://user-images.githubusercontent.com/102307923/205853063-b91c4567-e751-40c8-8944-ceebf61d6370.png)

**Comment by wldrve** [comment]
@itsalimanuel I know that, I've been working with Element UI and Element Plus now for a couple of years. The issue is, we use the "el-checkbox" component in conjuntion with Vue and Laravel. On the Login page (blade view from Laravel) we use the <el-checkbox> for "Remember me" checkbox and one has to only and the native "name=remember" attribute for it to work. It worked up until the Element Plus version 2.2.15 and I have just updated it to the latest and it does not work anymore. Either the "name" attribute or the "value" attribute changed somehow. 

This works with the native HTML <input type="checkbox" name="remember" ...>.

**Comment by itsalimanuel** [comment]
may this  [code](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG5jb25zdCBtc2cgPSByZWYoJ0hlbGxvIFdvcmxkIScpXG5jb25zdCBpbnB1dFJlZiA9IHJlZihmYWxzZSlcblxuY29uc3QgY2hhY2tWYWx1ZSA9IChpbnB1dFJlZikgPT4ge1xuXHRjb25zdCBpbnB1dCA9IGRvY3VtZW50LnF1ZXJ5U2VsZWN0b3IoJy5lbC1jaGVja2JveF9faW5wdXQgaW5wdXQnKVxuICBpbnB1dC52YWx1ZSA9IGlucHV0UmVmXG59XG5cbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gICAgPGVsLWNoZWNrYm94IEBjaGFuZ2U9XCJjaGFja1ZhbHVlKGlucHV0UmVmKVwiIHYtbW9kZWw9XCJpbnB1dFJlZlwiPmNoYW5nZTwvZWwtY2hlY2tib3g+XG4gIDxicj5cbiAgPHNwYW4+VmFsdWUgOiB7e2lucHV0UmVmfX08L3NwYW4+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319) help ?

**Comment by wldrve** [comment]
@itsalimanuel No, but thanks! I understand everything you did, you just have to know I do not want to add any script tags at the login page. The issue is el-checkbox component acts differently from the native checkbox, which is not great. I have no option of referencing the model value at the login page. :)

**Comment by wldrve** [comment]
UPDATE: I fixed my issue by adding ":label" attribute the same value as my v-model/model-value and hidding it. When v-model changes, so does the label attribute in el-checkbox and also the corresponding original input's "value" attribute. I then hide the .el-checkbox__label with CSS. 

Changing original input's "value" attribute with el-checkbox's "label" attribute is WRONG! The "value" in the original input should be the same as v-model/model-value not the "label".

This is an UGLY HACK and the EL-CHECKBOX component SHOULD BE FIXED/REFACTORED!!!

**Comment by Dsaquel** [comment]
>Original checkbox value should be the same as binding v-model

Since you has not explicitly binding the value attribute it is an expected behavior.

<img width="799" height="247" alt="Image" src="https://github.com/user-attachments/assets/06981a4f-3205-4aaa-85b2-fde2a934682e" />

The label is just an fallback if you're not setting `value`.
